dapp-developer-guide/tutorials/use-ethstorage-sdk-to-upload-and-download-files.md

```diff
@@ -15,80 +15,85 @@ You can install `ethstorage-sdk` by the following command:
 npm i ethstorage-sdk
 ```
 
-## Step 2: Upload Files
+## Step 2: Manage Files
 
+### 2.1 Create Flat Directory
+
+In this section, you will create a `FlatDirectory` contract for managing files.
 
 ```js
-const {BlobUploader, EncodeBlobs} = require("ethstorage-sdk");
+const {EthStorage} = require("ethstorage-sdk");
 
-const upload = async (filePath) => {
+const create = async () => {
+    const ETH_STORAGE_ADDRESS = "0x804C520d3c084C805E37A35E90057Ac32831F96f";
     const rpc = "https://rpc.sepolia.org";
     const privateKey = "0x...";
-    const tx = {
-      value: cost,
-      to: to,
-      data: data,
-    };
-    
-    const content = fs.readFileSync(filePath);
-    const blobs = EncodeBlobs(content);
-    const blobUploader = new BlobUploader(rpc, privateKey);
-    const hash = await blobUploader.sendTx(tx, [blobs[0], blobs[1]...) // max is 6
-    // hash : 0x37df32c7a3c30d3...52453dadacc838461d8629016
-}
-```
-
-## Step 3: Manage Files
+ 
+    const ethStorage = new EthStorage(rpc, privateKey);
+    await ethStorage.deploy(ETH_STORAGE_ADDRESS);
+    // FlatDirectory address: 0x37df32c7a3c30d352453dadacc838461d8629016
 
-### 3.1 Create Flat Directory
+    or
```

Comment:
I suggest moving the comments out of the code and using another paragraph and code sample.